core/client/fs/src/main/java/alluxio/client/file/MetadataCachingBaseFileSystem.java

```diff
@@ -227,4 +227,39 @@ public synchronized void close() throws IOException {
       super.close();
     }
   }
+
+  /**
+   * @param uri the uri need to drop metadata cache
+   * @return a mock RUIStatus object
+   */
+  public URIStatus dropMetadataCache(AlluxioURI uri) {
```

Comment:
I try to guess what @LuQQiu and @apc999 want is to make the `MetadataCachingBaseFileSystem.java` be more clean, so I suggest to make each of the new added method just return the `void` or long type cache size